index.go

```diff
@@ -225,3 +225,25 @@ type IndexBuilder interface {
 	Index(doc Document) error
 	Close() error
 }
+
+// IndexCopyable is an index which supports an online copy operation
+// of the index.
+type IndexCopyable interface {
+	// CopyTo creates a fully functional copy of the index at the
+	// specified destination directory implementation.
+	CopyTo(d Directory) error
+}
+
+type ReaderCopyable interface {
+	CopyableReader() CopyReader
+}
```

Comment:
Naming is quite funky :), also these need not really be exported - and should be relocated into bleve, alongside the IndexCopyable.

Reply:
ok